src/displayapp/screens/List.h

```diff
@@ -2,6 +2,7 @@
 
 #include <lvgl/lvgl.h>
 #include <cstdint>
+#include <array>
 #include <memory>
```

Comment:
I noticed that `<array>` is currently included through `<memory>`, which is otherwise unused, so `<memory>` should be removed.

Reply:
indeed, removed, compiled locally. Fixed now